lib/DDGC/DB/Result/InstantAnswer.pm

```diff
@@ -395,6 +395,53 @@ many_to_many 'users', 'instant_answer_users', 'user';
 has_many 'instant_answer_topics', 'DDGC::DB::Result::InstantAnswer::Topics', 'instant_answer_id';
 many_to_many 'topics', 'instant_answer_topics', 'topic';
 
+after insert => sub {
+    my ( $self ) = @_;
+    my $schema = $self->result_source->schema;
+    $schema->resultset('ActivityFeed')->create(
+        $self->ddgc->config->subscriptions->created_ia_page( {
+            meta1        => $self->id,
+            meta2        => join('', map { sprintf ':%s:', $_ }
+                $self->topics->columns([qw/ name /])->all),
```

Comment:
Hmm I wonder if you could put a bunch of this logic into the `$schema->resultset('ActivityFeed')->create_for_new_ia_page()` method I was suggesting above.  Not sure.